examples/config-multiple.json

```diff
@@ -0,0 +1,30 @@
+{
+    "version": "0.1.0-dev",
+    "storage": {
+        "rootDirectory": "/tmp/zot",
+        "dedupe": true,
+        "gc": true,
```

Comment:
what is the default value for gc/dedupe? I think default should be "true". I guess the only way we would say false here is if the underlying storage is actually shared.

Reply:
The approach so far has been zot always default to dist-spec, and extensions need to enabled explicitly.